lib/Resources/Driver.js

```diff
@@ -31,12 +31,12 @@ class Driver {
 
         await elements.forEach(async element => await element.submit());
 
-        this.logger.info(`Submit form "${selector}"`);
+        // this.logger.info(`Submit form "${selector}"`);
```

Comment:
I really like the simplicity of capturing the photo URLs printing the scripts output to file. The logger service (that uses Winston) has an option of logging to a file (with support for filters). We could keep these debug logs in and have a custom Winston label for writing the eventual image src to file.

maybe something like...

```
this.logger.info('a debug log not written to file');
this.logger.write('a log that is written to file');
```

If you'd rather keep this as it is, maybe we should remove any instances of these logs instead of commenting them out?